cli/server/server.go

```diff
@@ -107,11 +108,23 @@ func getConfigFromContext(ctx *cli.Context) (config.Config, error) {
 	return config.Load(configPath, net)
 }
 
-// handleLoggingParams enables debugging output is that's requested by the user.
-func handleLoggingParams(ctx *cli.Context) {
+// handleLoggingParams enables debugging output and logPath requested by the user .
```

Comment:
Hmm.

Reply:
" ." -> "." ?

Reply:
Yeah, but I also think it can be reworded a little because at the moment it reads like `enables ... logPath`, enabling log path doesn't sound right to me.